packages/transformer/src/BigMap.ts

```diff
@@ -0,0 +1,83 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { Id64String } from "@itwin/core-bentley";
+
+/**
+ * This class serves as a temporary solution to address the issue of the V8 JavaScript engine's Map implementation,
+ * which is limited to approximately 16.7 million elements. Currently, only valid or invalid Id64 string
```

Comment:
"valid or invalid" is confusing. How about:
```suggestion
 * which is limited to approximately 16.7 million elements. Currently, only Id64 string
```